src/protocol/notarization.md

```diff
@@ -0,0 +1,15 @@
+# Notarization
+
+As part of the TLSNotary protocol, the `Prover` can create authenticated commitments to the plaintext and have the `Notary` sign them without ever seeing the plaintext. This offers a way for the `Prover` to selectively prove the authenticity of arbitrary portions of the plaintext to a different `Verifier`. 
```

Comment:
We will have to introduce the notary here again, as a special case where.

I may also be better to move this chapter to the end of the protocol chapters.